shaping/shaper.go

```diff
@@ -74,7 +74,7 @@ func (t *HarfbuzzShaper) Shape(input Input) Output {
 	t.buf.Props.Language = input.Language
 	t.buf.Props.Script = input.Script
 	// TODO: figure out what (if anything) to do if this type assertion fails.
-	font := harfbuzz.NewFont(input.Face.(harfbuzz.Face))
+	font := harfbuzz.NewFont(input.Face)
```

Comment:
I guess the comment above now has no meaning and should be removed?

Reply:
Agreed

Reply:
Good point. I'll get that fixed :)

Reply:
Should be fixed now